include/service/entities/site_service.js

```diff
@@ -123,6 +142,16 @@ module.exports = function SiteServiceModule(pb) {
     };
 
     /**
+     * Get all of the redirect_hosts in the database
+     * @method getRedirectHosts
+     * @param {Function} cb - the callback function
+     */
+    SiteService.prototype.getRedirectHosts = function(cb) {
+      var dao = new pb.DAO();
+      dao.q(REDIRECT_HOSTS, { select: pb.DAO.SELECT_ALL, where: {} }, cb);
+    };
```

Comment:
New DB collection: "redirect_hosts". Is this the best way to retrieve this collection?